templates/deployment.yaml

```diff
@@ -630,7 +630,40 @@ spec:
         - name: pcap-data
           mountPath: /data/
       {{- end }}
-
+      {{- if .Values.rclone.enabled }}
+      ## Rclone ##
+      - name: rclone
+        image: {{ .Values.rclone.image.repository }}:{{ .Values.rclone.image.tag }}
+        volumeMounts:
+        - name: pcap-data
+          mountPath: /data/
+        imagePullPolicy: {{ .Values.rclone.image.pullPolicy | quote }}
+        ports:
+        - containerPort: 22
+          name: ssh-sftp
```

Comment:
Why do we have to declare an SSH port?

Reply:
as seen in production: sometimes sftp is listening on 2287. or 2247.

Reply:
But we don't listen - we're client. And SFTP is just one single option to push. I don't think we need any specific port declaration here.

Reply:
ah, `listening`. ja, its not about listening at all. except for pcaps.